images/win/scripts/Installers/Install-Firefox.ps1

```diff
@@ -4,17 +4,66 @@
 ################################################################################
 
 Import-Module -Name ImageHelpers -Force
-
 $temp_install_dir = 'C:\Windows\Installer'
 New-Item -Path $temp_install_dir -ItemType Directory -Force
 
-Install-EXE -Url "https://seleniumwebdrivers.blob.core.windows.net/knownfirefoxversion/FirefoxSetup.exe" -Name "FirefoxSetup.exe" -ArgumentList "-ms"
+$versionsJson = Invoke-WebRequest  "https://product-details.mozilla.org/1.0/firefox_versions.json" | ConvertFrom-Json
+$latestVersion = $($versionsJson.LATEST_FIREFOX_VERSION)
+Write-Host "Firefox latest version: $latestVersion"
+
+# url for latest version of firefox
+$urlLatestVersion = "https://download.mozilla.org/?product=firefox-latest&os=win64&lang=en-US"
```

Comment:
Since you get $latestVersion above, please use the link that point to the specific version instead of latest one

Reply:
Done.